OpenSim/Sandbox/MatlabScripts/matlabArrayToOpenSimTimeSeriesTable.m

```diff
@@ -0,0 +1,66 @@
+function table = matlabArrayToOpenSimTimeSeriesTableVec3(table_array)
+%% Matlab Utility to take a Matlab Struct and convert into an OpenSim Table
+
+
+%%
+import org.opensim.modeling.*
+% table_array = force_array
+
+%% make a empty table
+table = DataTable;
+
+%% get the label names from the structure
+labels = fieldnames(table_array);    
+
+%% Set the number of columns and the column labels
+colnames = StdVectorString;
+for i = 1 : length(labels)
+   
+    if ~strcmp(labels{i},'time')
+        colnames.add(labels{i})
+    end
+end
+
+table.setColumnLabels(colnames);
+
```

Comment:
It would be good to make sure all fields in `table_array` have the same number of entries.